backend/package.json

```diff
@@ -19,5 +19,8 @@
     "jsonwebtoken": "^8.5.1",
     "mongoose": "^6.5.2",
     "validator": "^13.7.0"
+  },
+  "devDependencies": {
+    "cross-env": "^7.0.3"
```

Comment:
We don't want that anymore don't we? :thinking:

Reply:
@atxr @amtoine

Reply:
tbh i'm not sure at all :eyes: 

do we need the dependency or not?

Reply:
Not at all

Reply:
then it should be removed :+1:

Reply:
yes it should!

Reply:
did it!